sdk/alertsmanagement/Azure.ResourceManager.AlertsManagement/src/Generated/Models/UnknownAlertProcessingRuleAction.cs

```diff
@@ -10,19 +10,19 @@
 
 namespace Azure.ResourceManager.AlertsManagement.Models
 {
-    /// <summary> Unknown version of Action. </summary>
-    internal partial class UnknownAction : AlertProcessingRuleAction
+    /// <summary> Unknown version of AlertProcessingRuleAction. </summary>
+    internal partial class UnknownAlertProcessingRuleAction : AlertProcessingRuleAction
```

Comment:
This renaming diff is because `rename-mapping`
```yaml
rename-mapping:
  Action: AlertProcessingRuleAction
```
With swagger input, Unknown types were generated before renaming. After converting to InputType from schema, Unknown types are generated after renaming.
Since this is internal, it's fine to keep the renaming.
And also, with TypeSpec input, we will get the input types with renaming names anyway.